### PR TITLE
Update events 1.10

### PR DESCRIPTION
**Pull request by vieux**
This PR should bring us one step closer to engine-master green in the CI

### CODE REVIEW

**Comment by vieux** [comment]
OMG

**Comment by vieux** [comment]
:green_apple: :green_book: :green_heart:

**Comment by abronan** [comment]
:tada: :balloon: LGTM.

**Comment by dongluochen** [comment]
LGTM.

**Comment by dongluochen** [comment]
Thank you Victor! :tada:

File: cluster/engine.go [hunk 1]
```diff
@@ -886,12 +892,12 @@ func (e *Engine) handler(ev *dockerclient.Event, _ chan error, args ...interface
 	case "die", "kill", "oom", "pause", "start", "stop", "unpause", "rename":
```
Comment (fugr) [inline cluster/engine.go]: why leave `container update` behind?

**Pull request merged**